debian_install.md

```diff
@@ -0,0 +1,100 @@
+# Installing st2 on debian/ubuntu
+
+Follow this guide and we'll lead through the basic installation and setup steps.
+
+## Setting up repos and fetching required software
+
+First let's update the system and fetch software which will be required during installation and setup process:
+```shell
+sudo apt-get update
+sudo apt-get install -y wget apache2-utils software-properties-common apt-transport-https sysvinit-utils
+```
+
+Second, add repositories and install st2.
+```shell
+## NB! This step provides instructions for ubuntu trusty
+# Don't forget to use correct distro name!
+export DISTRO=trusty
+
+wget -qO - https://bintray.com/user/downloadSubjectPublicKey?username=bintray | sudo apt-key add -
+
+sudo -E add-apt-repository "deb https://dl.bintray.com/stackstorm/${DISTRO}_staging stable main"
```

Comment:
For `wheezy` at least from my experiments `add-apt-repository` won't work, also `http` should be used to make it working.
So it's better just `>> /etc/apt/sources.list`